tests/integration/security/main_test.go

```diff
@@ -45,10 +45,30 @@ func setupConfig(ctx resource.Context, cfg *istio.Config) {
 	if cfg == nil {
 		return
 	}
-
+	// command to generate certificate
+	// openssl req -x509 -nodes -newkey rsa:2048 -keyout server.key -out server.crt -days 3650
```

Comment:
I don't think this command generates the following certificate, could you just mention to follow the `server.crt` instructions in `samples/jwt-server/testdata/README.md`

Reply:
cool. I'll do that.